app/navigators/BeforeYouStartNavigator.scala

```diff
@@ -72,4 +72,10 @@ class BeforeYouStartNavigator @Inject()(val dataCacheConnector: UserAnswersCache
         NavigateTo.dontSave(SessionExpiredController.onPageLoad())
     }
   }
+
+  private def navigateToTaskList() =
```

Comment:
This bit of code is repeated over and over again in controllers and views. Is there not somewhere common it could be put, e.g. in a template for the views and a base class for controllers? I know it will be removed after hub released but it would be much easier to remove if it were in once or two places.